apps/site/src/views/Home/components/StickerLayout/StickerLayout.tsx

```diff
@@ -0,0 +1,29 @@
+import { HackSticker, HeartSticker } from "@/components/Sticker/Stickers";
+
+export default function StickerLayout({
+	children,
+}: {
+	children: React.ReactNode;
+}) {
```

Comment:
Chore: we should extend from the `PropsWithChildren` type here.

Reply:
I just copied how it is in layout.tsx. Would it be ok to leave it the same?